src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -117,53 +131,57 @@ public Command rollerStop() {
         });
     }
 
-    public Command openArms() {
-        return new Command("Open Arms", this) {
-            @Override
-            protected void execute() {
-                pivotPointsMotor.set(.2);
-            }
+    public Command gamepieceCheck() {
+        // This command will pick up a ball
+        return new Command("Check for Gamepiece", this) {
 
             @Override
             protected boolean isFinished() {
-                return foldedBackLimitSwitch.get();
-            }
-
-            @Override
-            protected void end() {
-                pivotPointsMotor.set(0);
+                return gamepieceLimitSwitch.get();
             }
         };
     }
+    // #endregion
+
+    public Command Intake() {
+        return new Command("Intake Ball", this) {
 
-    public Command closeArms() {
-        return new Command("Close Arms", this) {
             @Override
-            protected void execute() {
-                pivotPointsMotor.set(-.2);
+            protected void initialize() {
+                armsPiston.set(Value.kForward);
+                rollersMotor.set(rollerspeed);
             }
 
             @Override
             protected boolean isFinished() {
-                return intakePositionLimitSwitch.get();
+                return false;
             }
 
             @Override
             protected void end() {
-                pivotPointsMotor.set(0);
+                armsPiston.set(Value.kReverse);
+                rollersMotor.set(0);
             }
         };
     }
 
-    public Command gamepieceCheck() {
-        // This command will pick up a ball
-        return new Command("Check for Gamepiece", this) {
+    public TimedCommand Eject() {
+        return new TimedCommand("Eject Ball", 1) {
```

Comment:
`return new TimedCommand("Eject Ball", 1, this)` - you need to add `this` otherwise it'll cause scheduler problems

Reply:
Thank you, done